src/testing/test_runner.cc

```diff
@@ -66,7 +66,11 @@ void test_runner::start_thread(int ac, char** av) {
 
     namespace bpo = boost::program_options;
     _thread = std::make_unique<posix_thread>([this, ac, av, init_outcome]() mutable {
-        app_template app;
+        app_template::config cfg;
+        if (::getenv("SEASTAR_NATIVE_TEST") != nullptr && std::string(::getenv("SEASTAR_NATIVE_TEST")) == "yes") {
+            cfg.use_scheduling_groups_as_io_classes = true;
```

Comment:
I don't think relying on environment variables is a good idea. You don't **have** to run tests via cmake. I often run a single test by just running the single executable - now this won't work?

I think that the solution should be different: Most Seastar tests don't care at all if this flag is enabled or disabled, so it can be left the default (disabled, Avi asked for enabled). Only the very few tests that are related to this flag should set this flag explicitly, in the individual test's code.